uart-msgpacker/src/test/java/nl/isaac/androidlibs/uart_msgpacker/RequestModelsTest.kt

```diff
@@ -16,7 +16,7 @@ class RequestModelsTest {
         val rid = 2
         val request = ReadRequest(rid, modbusIdList.toTypedArray())
         // Pack request
-        val packedRequest = request.packRequest()
+        val packedRequest = request.packRequest(shouldUseNewKeys())
```

Comment:
Where is this 'shouldUseNewKeys' method located? I can't see it in the changes in this MR?

Reply:
removed